plugin/ochttp/client.go

```diff
@@ -57,6 +57,9 @@ type Transport struct {
 // RoundTrip implements http.RoundTripper, delegating to Base and recording stats and traces for the request.
 func (t *Transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	rt := t.base()
+	if req.URL.Path == "/healthz" {
```

Comment:
Thanks for the quick fix @rakyll!
 
Could we please add a comment for the rationale of this change?
```go
// Disable tracing /healthz since tracing it is costly and
// it bloats Kubernetes traces without adding much value
```

Reply:
Done.